runtime/examples/tapasco-debug/debug-screens/BlueDebugScreen.hpp

```diff
@@ -256,4 +256,4 @@ class BlueDebugScreen : public MenuScreen {
   platform_ctl_addr_t intc_addr;
 };
 
-#endif
\ No newline at end of file
+#endif
```

Comment:
In a separate commit please.

Reply:
Rebasing seems impossible at this point so I don't think I can do this.